pr-author.md

```diff
@@ -0,0 +1,14 @@
+# Pull request submission workflows
+
+## Workflow for pull request author
+
+```mermaid
+graph TD
+    A[I want to update MDN Web Docs]-->B{Is there an <br> issue or discussion <br> that explains these changes?}
```

Comment:
```suggestion
    A[I want to update MDN Web Docs]-->B{Is there an <br> issue or discussion <br> that explains these changes?}
```
The decision block ideally also needs to convey that if the PR is self-explanatory and won't require/lead to a discussion, it might be okay to open a PR directly.